src/create-reminder-signup/lib/db.ts

```diff
@@ -8,29 +8,30 @@ export function writeOneOffSignup(
 ): Promise<QueryResult> {
 	const query: QueryConfig = {
 		text: `
-            INSERT INTO one_off_reminder_signups(
-                identity_id,
+			INSERT INTO one_off_reminder_signups(
+				identity_id,
 				country,
-                reminder_period,
-                reminder_created_at,
-                reminder_platform,
-                reminder_component,
-                reminder_stage,
-                reminder_option
-            ) VALUES (
-                $1, $2, $3, $4, $5, $6, $7, $8
-            )
-            ON CONFLICT ON CONSTRAINT one_off_reminder_signups_pkey
-            DO
-                UPDATE SET
-                    country = $2,
-                    reminder_created_at = $4,
-                    reminder_platform = $5,
-                    reminder_component = $6,
-                    reminder_stage = $7,
-                    reminder_option = $8
-            RETURNING *;
-        `,
+				reminder_period,
+				reminder_created_at,
+				reminder_platform,
+				reminder_component,
+				reminder_stage,
+				reminder_option
+			) VALUES (
+				$1, $2, $3, $4, $5, $6, $7, $8
+			)
+			ON CONFLICT ON CONSTRAINT one_off_reminder_signups_pkey
+			DO
+				UPDATE SET
+					country = $2,
+					reminder_created_at = $4,
+					reminder_platform = $5,
+					reminder_component = $6,
+					reminder_stage = $7,
+					reminder_option = $8,
+					reminder_cancelled_at = NULL
+			RETURNING *;
+		`,
```

Comment:
It's slightly hard to see what's happening here. 

Unimportant = all spaces -> tabs
Important = set reminder_created_at = NULL when overwritting the reminder

I almost missed this whilst implementing. It's tempting to maybe write some tests around this.

Reply:
Just to add here - this would mean adding tests to the `create-reminder-signup` db queries, which we don't yet have. If we think that's a good idea, we I think we do it in a follow up PR

Reply:
good spot